README.md

```diff
@@ -98,6 +99,18 @@ const [data, setString] = useClipboard()
 <Button title='Update Clipboard' onPress={() => setString('new clipboard data')}>Set Clipboard</Button>
 ```
 
+### `useDeepLinking`
+
+> __Note:__ You'll still need to properly enable deep linking within your application, here are the [steps](https://reactnative.dev/docs/linking#enabling-deep-links)
+
+```js
+import { useDeepLinking } from '@react-native-community/hooks'
+
+useDeepLinking((url) => {
+  // handle deep linking logic
```

Comment:
Could you add an actual example here? That it would be easier for us to test before merging, and for users in the future to get an idea of what to put in there. Thank you. 🙏

Reply:
I added an example, I hope its good enough. Please let me know :)